api/openapi.yaml

```
@@ -41,32 +41,77 @@ components:
+              - type: object
+                properties:
+                  error:
+                    type: string
+                    example: "invalid_request"
+                  error_description:
+                    type: string
+                    example: "Authorization header value must follow this format: Bearer access-token"
+                  message:
+                    type: string
+                    example: "Bad credentials"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)